Octokit/Models/Response/PullRequest.cs

```diff
@@ -34,7 +34,7 @@ public PullRequest(Uri url, Uri htmlUrl, Uri diffUrl, Uri patchUrl, Uri issueUrl
             Base = @base;
             User = user;
             MergeCommitSha = mergeCommitSha;
-            Merged = merged;
+            Merged = merged || MergedAt.HasValue;
```

Comment:
Let's completely remove the `merged` constructor parameter. `Merged` will be a calculated property. So this should be `Merged = MergedAt.HasValue`.